okta-hosted-login/README.md

```diff
@@ -14,9 +14,20 @@ Before running this sample, you will need the following:
 
 Clone this repo and replace the okta configuration placeholders in the `appsettings.json` with your configuration values from the Okta Developer Console. 
 You can see all the available configuration options in the [okta-aspnet GitHub](https://github.com/okta/okta-aspnet/blob/master/README.md).
-For step-by-step instructions, visit the Okta [ASP.NET MVC quickstart]. The quickstart will guide you through adding Okta login to your ASP.NET application.
+For step-by-step instructions, visit the Okta [Sign Users in to Your Web Application guide]. The guide will walk you through adding Okta login to your ASP.NET application.
 
-Now start your server and navigate to http://localhost:8080 in your browser.
+> **NOTE:** This sample is using ASP.NET Core 2.2 which enforces HTTPS. This is a recommended practice for web applications. Check out [Enforce HTTPS in ASP.NET Core] for more details.
+
+#### Visual Studio
+
+If run this project in Visual Studio it will start the web application on ports 5000 for HTTP and 44314 for HTTPS. You can change this configuration in the `launchSettings.json`. Make sure to [update your Okta Application] in your Developer Console with the correct Base URI (for example https://localhost:44314).
```

Comment:
You sure it's 5000 for HTTP? I thought IIS Express was always >40000.

Reply:
Yep, I decided to use 5000 for HTTP for both options (Kestrel and IIS). The restriction you mention is only for HTTPS. Actually, we used to run this sample in the port 8080, we can keep this port for IIS if you think it makes more sense. I didn't want to add more confusion by using a different port but it has some cons and pros. For example, if we keep in mind that all our samples are currently running on 8080 it would make more sense to keep using 8080.
On the other hand, by using the same HTTP port for both running options (dotnet vs VS) we give devs the opportunity to use the same Okta settings regardless of how the run the project, but this is only valid if they omit HTTPS.